Code/Framework/AzCore/AzCore/EBus/EventSchedulerSystemComponent.cpp

```diff
@@ -109,7 +109,7 @@ namespace AZ
             timedEvent->m_handle = AllocateHandle();
         }
         const bool ownsScheduledEvent = false;
-        *(timedEvent->m_handle) = ScheduledEventHandle(TimeMs(currentMilliseconds + durationMs), durationMs, timedEvent, ownsScheduledEvent);
+        timedEvent->m_handle = new (timedEvent->m_handle) ScheduledEventHandle(TimeMs(currentMilliseconds + durationMs), durationMs, timedEvent, ownsScheduledEvent);
```

Comment:
Ah, so this was using the copy-assignment operator on an uninitialized instance of the handle, which would attempt to use the uninitialized to unhook the old handle?